### ISSUE TITLE
Accessibility: Step title label is not associated with content

### ISSUE THREAD

**Issue by zhr1130** [react-component/steps#153]
We are using radio buttons as the step content and we got accessibility bugs since the label is not associated with the radio button. 
![image](https://user-images.githubusercontent.com/18317674/120393474-d4007b00-c2e6-11eb-888f-78bab1fb5b77.png)
![image](https://user-images.githubusercontent.com/18317674/120393502-debb1000-c2e6-11eb-8906-ec203b6810e8.png)
